symengine/parser.h

```diff
@@ -7,6 +7,7 @@ namespace SymEngine
 {
 
 RCP<const Basic> parse(const std::string &s, bool convert_xor = true);
```

Comment:
Can you add the local_parser_constants argument here and it pass it through?

Reply:
Yes, done.